dnsdist: Fails to build with Boost 1.70 (master snapshot)

Regressed by boostorg/smart_ptr@eac641186738. Found on FreeBSD on dnsdist-1.3.3. May not affect other platforms due to header botlegging.

```c++
dnsdist-lua-vars.cc:99:61: error: expected unqualified-id
                                       {"BADSIG",   ERCode::BADSIG  },
                                                            ^
/usr/include/sys/signal.h:490:17: note: expanded from macro 'BADSIG'
#define BADSIG          SIG_ERR
                        ^
/usr/include/sys/signal.h:139:18: note: expanded from macro 'SIG_ERR'
#define SIG_ERR         ((__sighandler_t *)-1)
                        ^
1 error generated.
```

--- a/pdns/dnsdistdist/html/local.js
+++ b/pdns/dnsdistdist/html/local.js
@@ -194,7 +194,8 @@ $(document).ready(function() {
                      var bouw='<table width="100%"><tr align=right><th>#</th><th align=left>Name</th><th align=left>Address</th><th>Status</th><th>Latency</th><th>Queries</th><th>Drops</th><th>QPS</th><th>Out</th><th>Weight</th><th>Order</th><th align=left>Pools</th></tr>';
                      $.each(data["servers"], function(a,b) {
                          bouw = bouw + ("<tr align=right><td>"+b["id"]+"</td><td align=left>"+b["name"]+"</td><td align=left>"+b["address"]+"</td><td>"+b["state"]+"</td>");
-                         bouw = bouw + ("<td>"+(b["latency"]).toFixed(2)+"</td><td>"+b["queries"]+"</td><td>"+b["reuseds"]+"</td><td>"+(b["qps"]).toFixed(2)+"</td><td>"+b["outstanding"]+"</td>");
+                         var latency = (b["latency"] === null) ? 0.0 : b["latency"];
+                         bouw = bouw + ("<td>"+latency.toFixed(2)+"</td><td>"+b["queries"]+"</td><td>"+b["reuseds"]+"</td><td>"+(b["qps"]).toFixed(2)+"</td><td>"+b["outstanding"]+"</td>");
                          bouw = bouw + ("<td>"+b["weight"]+"</td><td>"+b["order"]+"</td><td align=left>"+b["pools"]+"</td></tr>");
                      }); 
                      bouw = bouw + "</table>";

